src/test/java/racingcar/controller/GameControllerTest.java

```diff
@@ -39,29 +40,29 @@ class GameControllerTest {
     @Test
     @DisplayName("post(/plays) 테스트")
     void post_plays() throws Exception {
-        final List<Car> cars = List.of(new Car("aa"));
-        given(gameService.createGameWith(any(), anyInt()))
-                .willReturn(new Game(cars, 10));
+        final List<CarDto> cars = List.of(new CarDto("aa", 1));
+        given(gameService.playWith(any(), anyInt()))
+                .willReturn(new ResultDto(List.of("aa"), cars));
```

Comment:
```suggestion
                .willReturn(new ResultDto(List.of("땡칠"), cars));
```

테스트 코드지만, 좀 더 의미있는 값을 쓰면 좋을 것 같아요 ㅎㅎ

Reply:
반영했습니다.
어떤 값이든 들어갈 수 있는 테스트 데이터라도, 구체적으로 표현되어 있는 편이 읽는 입장에서 좋겠네요.